mantis-client/src/main/java/io/mantisrx/client/MantisSSEJob.java

```diff
@@ -179,10 +180,6 @@ public static class Builder {
         private Observer<SinkConnectionsStatus> sinkConnectionsStatusObserver = null;
         private long dataRecvTimeoutSecs = 5;
 
-        public Builder(Properties properties) {
-            this(new MantisClient(properties));
-        }
-
         public Builder() {
```

Comment:
We were seeing log spam, where a JobSource triggers this code path.  It doesn't appear to break anything, but seems a bit weird and leads to log spam (`HighAvailabilityServicesUtil` logs a few warnings, because this path attempts to "create" the HA services with no properties).

My understanding is that the HA services should generally be configured if we are here, so should just try to get the reference to HA services.